README.md

```diff
@@ -108,6 +108,18 @@ have to enter your Seqera credentials once.
 - **Token**: Your Seqera API token. Create a Seqera access token via [_Your Tokens_](https://cloud.seqera.io/tokens) in the user menu.
 - **Workspace ID**: Your Seqera workspace ID
 
+### Required Token Permissions
+
+| Node                      | Minimum Role Required |
+| ------------------------- | --------------------- |
+| Launch workflow           | Launch                |
```

Comment:
According to https://github.com/seqeralabs/node-red-seqera/issues/3
> This came up because a `launch` user wasn't able to launch a pipeline, it required `maintain`.

Which one is it?

```suggestion
| Launch workflow           | Maintain              |
```